kura/distrib/src/main/ant/build_equinox_distrib.xml

```diff
@@ -1391,6 +1431,28 @@ fi]]>
 	    </zip>
 	</target>
 
+    <target name="tinyb-snapshot-jars-all" if="is.tinyb.snapshot">
```

Comment:
Why this?

Reply:
relates to my comment above, we need to include all the architectures as we want to be able to build snap for armhf/arm64/amd64 from single target. 
Unless you prefer to have separate target for each architecture. Choice is yours :)